perftest_qos_profiles.xml

```diff
@@ -250,8 +250,8 @@ _KeepDurationUsec variable name corresponds to the command-line option -keepDura
 
       </participant_qos>
 
-      <!-- === Publisher Configuration: ================================= 
-        
```

Comment:
If we are finally not changing this file, could you revert all the space changes?